### PR TITLE
feat: moved apply license form to antd form

### PR DESCRIPTION
**Pull request by GermaVinsmoke**
Solves feature issue - invite member form of https://github.com/SigNoz/signoz/issues/2150

### CODE REVIEW

File: frontend/src/container/Licenses/ApplyLicenseForm.tsx [hunk 1]
```diff
@@ -66,16 +70,19 @@ function ApplyLicenseForm({
...
-					/>
+			<ApplyForm
+				form={form}
+				layout="inline"
+				onFinish={onFinish}
+				autoComplete="off"
+			>
+				<LicenseInput
+					name="key"
+					rules={[{ required: true, message: 'Missing License Key' }]}
```
[the remark this answers is not in the document]
Comment (GermaVinsmoke) [reply frontend/src/container/Licenses/ApplyLicenseForm.tsx:81]: Yeah, now it'll be possible

[the remark this answers is not in the document]
Comment (palashgdev) [reply frontend/src/container/Licenses/ApplyLicenseForm.tsx:81]: listed in this [one](frontend/src/utils/form/requireErrorMessage.ts)

[the remark this answers is not in the document]
Comment (GermaVinsmoke) [reply frontend/src/container/Licenses/ApplyLicenseForm.tsx]: Updated

File: frontend/src/container/Licenses/ApplyLicenseForm.tsx [hunk 2]
```diff
@@ -66,16 +71,19 @@ function ApplyLicenseForm({
...
+					name="key"
+					rules={[{ required: true, message: requireErrorMessage('License Key') }]}
+				>
+					<Input placeholder={t('placeholder_license_key')} />
 				</LicenseInput>
-				<FormItem>
+				<Form.Item>
 					<Button
 						loading={loading}
 						disabled={loading}
```
Comment (palashgdev) [inline frontend/src/container/Licenses/ApplyLicenseForm.tsx:89]: maybe disabling the form when the key length is zero ?

Comment (GermaVinsmoke) [reply frontend/src/container/Licenses/ApplyLicenseForm.tsx:89]: Alright

Comment (GermaVinsmoke) [reply frontend/src/container/Licenses/ApplyLicenseForm.tsx:89]: Done in #2775

**Pull request merged**